app/utils/auth.py

```
@@ -1,134 +1,176 @@
 """
 Clerk authentication utility for the DevDox AI Portal API.
 """
+
 import logging
 from dataclasses import dataclass, fields
-from typing import Any, ClassVar, Dict
+from typing import Any, ClassVar, Dict, Mapping, Optional, Protocol
```

Ruff (0.11.9)

`typing.Mapping` imported but unused

Remove unused import: `typing.Mapping`

(F401)